src/UIState/SeePIDConstants.cpp

```diff
@@ -4,12 +4,68 @@
 
 #include "SeePIDConstants.h"
 
-void SeePIDConstants::handleKey(char key) {
-  switch (key) {
-    case 'D':  // Don't save (cancel)
-      returnToMainMenu();
+#include "Devices/LiquidCrystal_TC.h"
+#include "Devices/PHProbe.h"
+#include "Devices/PID_TC.h"
+#include "Serial_TC.h"
+#include "TC_util.h"
+
+void SeePIDConstants::loop() {
+  int elapsedSeconds = (millis() - startTime) / 1000;
+
+  Serial_TC *mySerial = Serial_TC::instance();
+  mySerial->print("SeePIDConstants::loop() - 1 CurrentTime ", elapsedSeconds);
+  switch (elapsedSeconds) {
+    case 0:
+      loadKp(0);
+      loadKi(1);
+      break;
+    case 1:
+      loadKi(0);
+      loadKd(1);
+      break;
+    case 2:
+      loadKd(0);
+      loadSlope(1);
       break;
-    default:
+    case 3:
+      loadSlope(0);
+      loadKp(1);
       break;
-  };
+    case 4:
+      loadKp(0);
+      loadKi(1);
+      break;
+  }
+  if (elapsedSeconds >= 5) {
```

Comment:
Could this be a default case in the switch statement?

Reply:
With the way things are currently, having it in it's own if gives no benefit, so you're right a default would do the same thing.